tests/ui/attributes/mixed_export_name_and_no_mangle.rs

```diff
@@ -0,0 +1,10 @@
+// issue: rust-lang/rust#47446
+
+#![deny(mixed_export_name_and_no_mangle)]
```

Comment:
I would prefer for the test to use the default lint level, so if it changes it's reflected in the test.
```suggestion
//@ check-pass
```